modules/installation-gcp-shared-vpc-permissions.adoc

```diff
@@ -0,0 +1,21 @@
+// This file is referenced in the following assembly:
+// installing/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-permissions_{context}"]
+= Required GCP permissions for shared VPC installation
+
+When you attach the `Owner` role to the service account that you create, you
+grant that service account all permissions, including those that are required to
+install {product-title}. To deploy an {product-title} cluster into a shared VPC, the
+service account requires the following permissions in the host project.
+
+.Required roles for creating network resources during installation
+* Compute Network User
```

Comment:
The required roles/permissions we'd ever discussed in the original [PR](https://github.com/openshift/openshift-docs/pull/51171):

**Scenario A: Don't want the installation program to automatically create firewall rules for your cluster**
- the role `roles/compute.networkUser`
- the permission `dns.networks.bindPrivateDNSZone` [1]

**Scenario B: Want the installation program to automatically create firewall rules for your cluster** [2]
- the role `roles/compute.networkAdmin`
- the role `roles/compute.securityAdmin`
- the permission `dns.networks.bindPrivateDNSZone`

[1] https://github.com/openshift/openshift-docs/pull/51171#issuecomment-1277301267
[2] [Installing a cluster on GCP into a shared VPC](https://docs.openshift.com/container-platform/4.12/installing/installing_gcp/installing-gcp-shared-vpc.html#installation-gcp-shared-vpc-config_installing-gcp-shared-vpc)